File: Java/T6/codigos2122/JSON/app/src/main/java/com/example/json/SecondActivity.java
package com.example.json;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ImageView;

import com.bumptech.glide.Glide;
import com.example.json.utils.Equipo;

public class SecondActivity extends AppCompatActivity {

    private ImageView imagen;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);
        imagen = findViewById(R.id.imagen_recepcion);

        if (getIntent().getExtras()!=null){
            Equipo equipo = (Equipo) getIntent().getExtras().getSerializable("equipo");
            Glide.with(getApplicationContext()).load(equipo.getEscudo()).placeholder(R.drawable.escudo).into(imagen);
        }

    }
}

File: Kotlin/T2/codigos_ces/app/src/main/java/com/example/formulario/ui/activity/SecondActivity.kt
package com.example.formulario.ui.activity

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.formulario.R
import com.example.formulario.databinding.ActivityMainBinding
import com.example.formulario.databinding.ActivitySecondBinding

class SecondActivity : AppCompatActivity() {
    private lateinit var binding: ActivitySecondBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivitySecondBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}